cumulusci/cumulusci.yml

```diff
@@ -259,7 +259,7 @@ tasks:
             command: "force:source:pull"
         group: Salesforce DX
     dx_push:
-        description: Uses sfdx to push the force-app directory metadata into a scratch org
+        description: "DEPRECATED: Use the `deploy` command to deploy metadata instead. Use the `dx` command if you nned to run `force:source:push` for some reason."
```

Comment:
```suggestion
        description: "DEPRECATED: Use the `deploy` command to deploy metadata instead. Only use the `dx` command if you need to run `force:source:push` for some reason."
```

Reply:
The typo is fixed in 61abe4dc3.

The bit about "Only use" is probably based on a misunderstanding.